tutorials/npm/src/ts/data-provider-manager.ts

```diff
@@ -0,0 +1,87 @@
+import {
+    ICU4XDataProvider,
+    ICU4XLocale,
+    ICU4XLocaleFallbacker,
+} from 'icu4x';
+
+export class DataProviderManager {
+
+    private dataProvider: ICU4XDataProvider;
+    private loadedLocales: Set<ICU4XLocale>;
+
+    private constructor() {
+        this.loadedLocales = new Set<ICU4XLocale>();
+    }
+
+    public static async create(): Promise<DataProviderManager> {
+        const manager = new DataProviderManager();
+        await manager.init();
+        return manager;
+    }
+
+    private async init() {
+
+        const enFilePath = 'dist/en.postcard';
+        let enProvider = await this.createDataProviderFromBlob(enFilePath);
+        this.loadedLocales.add(ICU4XLocale.create_from_string("en"));
+        const unFilePath = 'dist/en.postcard';
```

Comment:
Tried this with `dist/und.postcard`, with no help. Will fix this in consecutive commits**